### PR TITLE
FEAT: adjust flex direction

### PR DESCRIPTION
**Pull request by ghgamble**
BEFORE | AFTER | &nbsp;
-- | -- | --
01-Screen Shot 2021-06-30 at 3 37 06 PM | 02-Screen Shot 2021-06-30 at 3 36 47 PM | Stack search bar under tabs and center on sm screens
03-Screen Shot 2021-06-30 at 3 37 30 PM | 04-Screen Shot 2021-06-30 at 3 37 18 PM | No visual changes on lg screens

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ export default function IndexPage({ pkgVersion }) {
             />
           </div>
         </div>
-        <div className="flex justify-between items-start shadow-banner px-4 lg:flex-row sm:flex-col sm:px-6 lg:px-6 bg-white">
+        <div className="flex justify-between items-start shadow-banner px-4 lg:flex-row sm:flex-col sm:px-6 lg:px-6 bg-white flex-col sm:flex-row">
```
Comment (ynotdraw) [inline src/pages/index.js:166]: Looks like you have two conflicting styles here. There are two setting the flex direction sm:flex-col and on the end now sm:flex-row

**Review by ynotdraw — CHANGES_REQUESTED** [review]

**Comment by ynotdraw** [comment]
Should the search field be full width as well when on mobile?  There's a `w-full` class you can use if so.

**Comment by dexterca** [comment]
I just pulled this down and SearchField is still not full width on mobile. I troubleshot this myself and found it incredibly difficult to fix (probably b/c I'm a Tailwind noob) but I think I have a fix now. @ghgamble let me know if you want to try fixing yourself or if you want me to commit my fix to this PR.

Screen Shot 2021-06-30 at 1 11 10 PM

**Comment by ghgamble** [comment]
That's fine with me.. I struggled to find a fix too. If you want to push it to this branch/PR. Thanks!

**Comment by ghgamble** [comment]
Casey was able to make a fix.

**Review by ynotdraw — APPROVED** [review]

**Pull request merged**